lib/Sanger/CGP/CavemanPostProcessing.pm

```diff
@@ -304,6 +304,9 @@ sub _callbackTumFetch{
 			$str = -1;
 		}
 		return unless ($algn->proper_pair == 1);
+    # Ensure that we keep 
+    return if((int($flagValue) & 16) != 0 && (int($flagValue) & 32) != 0);
```

Comment:
Can you not handle both conditions in one statement?

```
return if((int($flagValue) & 16) != 0 (int($flagValue) & 32));
```
Both can't be true, both can't be false